openstudiocore/src/openstudio_app/OpenStudioApp.cpp

```diff
@@ -1390,7 +1390,7 @@ std::vector<openstudio::path> OpenStudioApp::libraryPaths() const {
   QSettings settings(QCoreApplication::organizationName(), QCoreApplication::applicationName());
 
   openstudio::path resPath = resourcesPath();
-
+  std::cout << "resourcesPath = " << resPath << std::endl;
```

Comment:
@jmarrec these couts are going to go away before merging right?

Reply:
Done